lib/common/schemas.c

```diff
@@ -650,6 +652,51 @@ validate_xml(xmlNode *xml_blob, const char *validation, gboolean to_logs)
 }
 
 #if HAVE_LIBXSLT
+
+/* see crm_xml_error */
+static void
+crm_xslt_err(void *ctx, const char *msg, ...)
+G_GNUC_PRINTF(2, 3);
+
+static void
+crm_xslt_err(void *ctx, const char *msg, ...)
+{
+    int len = 0;
+    va_list args;
+    char *buf = NULL;
+    static int buffer_len = 0;
+    static char *buffer = NULL;
+
+    va_start(args, msg);
+    len = vasprintf(&buf, msg, args);
+
+    if (strchr(buf, '\n')) {
```

Comment:
I am now making it explicit in the log that there's some kind of corruption
so it makes sense that the previously logged chunk does not make sense alone.

BTW. it was a brilliant idea not to originally mention why is this dancing
around `\n` needed, indeed.